prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,14 @@
+const languages = ['html', 'css', 'javascript', 'react', 'node', 'postgres'];
+console.log(languages);
+var declareFirstElement = 'The first element of the array is';
+var firstElement = 'html';
+console.log(declareFirstElement, firstElement);
+var declareThirdElement = 'The 3rd element of the array is';
+var thirdElement = 'javascript';
+console.log(declareThirdElement, thirdElement);
+var declareLength = 'The length of the array is';
+var length = 6;
+console.log(declareLength, length);
+var declareLastElement = 'The last element of the array is';
+var lastElement = 'postgres';
```

Comment:
Here we want to use the `languages` array and get its last element. To figure out the last index we use the arrays `length` minus `1`